pyaedt/conftest.py

```diff
@@ -10,3 +20,14 @@ def start_aedt():
     # Wait to run doctest on docstrings
     yield desktop
     desktop.force_close_desktop()
+
+@pytest.fixture(autouse=True, scope="session")
+def get_root_directory(doctest_namespace):
+    """Setup paths to the root and project directories.
```

Comment:
```suggestion
    """Set up paths to the root and project directories.
```
I had to look this up myself. Please see [Setup vs set up](https://www.enhancemywriting.com/setup-vs-set-up).